test/BUILD.gn

```diff
@@ -514,6 +514,7 @@ if (!is_android) {
       "//brave/browser/extensions/brave_extension_provider_browsertest.cc",
       "//brave/browser/extensions/brave_theme_event_router_browsertest.cc",
       "//brave/browser/farbling/brave_enumeratedevices_farbling_browsertest.cc",
+      "//brave/browser/farbling/brave_navigator_devicememory_farbling_browsertest.cc",
```

Comment:
We no longer add tests directly to test/BUILD.gn. New tests need to go to their own source_set in the directory where they live and then added as a dep here.

Reply:
latest push has a `source_set("browser_tests")` in `browser/farbling/BUILD.gn`